### ISSUE TITLE
Internal: PR Templates (all METplus component repos): Improve language about linking relevant issue(s)

### ISSUE THREAD

**Issue by DanielAdriaansen** [dtcenter/METplus#2257]
## Describe the Enhancement ##
In the GitHub template for pull requests (PR's), there is a checkbox in the Pull Request Checklist that says:

`- [ ] After submitting the PR, select Development issue with the original issue number.`

We would like to modify this to say:

`- [ ] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.`

This is located in the PR template file, in most (or all) of the METplus components repositories. It can be found in each repository in the `.github/pull_request_template.md` file.  

The PR templates that are in the current default branch are used by GitHub.  **Be sure to apply all changes to the current default branch and also in the develop branch since the contents of that will eventually become the default branch in the future.**

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [x] METplus
- [x] MET (Make sure to use branch "main_v11.1" instead of "develop")
- [x] METcalcpy
- [x] METdataio
- [x] METplotpy
- [x] METviewer
- [x] METexpress
- [x] METplus-Training
- [x] METbaseimage

### Relevant Deadlines ###
None.

### Funding Source ###
Ask Tara before starting on this issue.

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by jprestop** [comment]
While I added this comment to this issue:

> Be sure to apply all changes to the current default branch and also in the develop branch since the contents of that will eventually become the default branch in the future.

Please note that to save ourselves some work we decided to only commit these changes into the develop branches of the repositories because we are only a week away from the release and these develop (main_v11.1 for MET, main for METbaseimage, development for METexpress) will become the main branches.   

Unfortunately, @lisagoodrich was unable to commit changes to the develop branch for other repos besides METplus (not sure why after looking at the settings for collaborators), so I helped make these changes in the other repositories.  This work is complete with no PRs because commits were made directory in develop (main_v11.1 for MET, main for METbaseimage, development for METexpress).
